core/internal/features_ocr2_test.go

```diff
@@ -106,7 +106,7 @@ func setupNodeOCR2(
 	config.Overrides.FeatureOffchainReporting = null.BoolFrom(false)
 	config.Overrides.FeatureOffchainReporting2 = null.BoolFrom(true)
 	config.Overrides.FeatureLogPoller = null.BoolFrom(true)
-	poll := 500 * time.Millisecond
+	poll := 5000 * time.Millisecond
```

Comment:
I don't think there's much value in polling super fast in the OCR integration test since the networking setup is slow anyways, will just load/thrash